linalg/hypre.hpp

```diff
@@ -525,6 +525,13 @@ class HypreParMatrix : public Operator
        Ae sum to the original matrix. */
    HypreParMatrix* EliminateRowsCols(const Array<int> &rows_cols);
 
+   /** Eliminate columns from the matrix and store the eliminated
+       elements in a new matrix Ae (returned) so that the modified
+       matrix and Ae sum to the original matrix. */
+   HypreParMatrix* EliminateCols(const Array<int> &cols);
+
+   void EliminateRows(const Array<int> &rows);
```

Comment:
This is missing a function description. It's pretty obvious what it does, but I think it's nice to include for completeness.